README.md

```diff
@@ -81,20 +147,33 @@ Suppose you're encoding geometric shapes in a Cartesian (2D) coordinate system,
 **a)**
 Write an expression for the rectangle's area.
 
+var area = (t - b) * (r - l);
+
 **b)**
 Write an expression which is true if the rectangle is taller than it is wide, and false otherwise.
 
+(t - b) > (r - l);
+
 **c)**
 Write an expression for the circumference of the biggest circle which can fit inside the rectangle.  (Hint: you'll need logic similar to that in **b**.)
 
+(t - b) > (r - l) ? 2 * 3.14 * ((r - l) * 0.5) : 2 * 3.14 * ((t - b) * 0.5); 
+
 **d)**
 Write an expression for the area of the smallest circle which completely encloses (i.e. circumscribes) the rectangle.
 
+(t - b) > (r - l) ? 2 * 3.14 * ((t - b) * 0.5) : 2 * 3.14 * ((r - l) * 0.5);
+
 **e)**
 Imagine subdividing your rectangle into 3 equal rows and 3 equal columns, which would create 9 smaller rectangles, identical in shape but varying by position.
 Define four new variables describing the centermost small rectangle.
 (_Hint: one of the many solutions is very similar to the solution of **2b** above._)
 
+var tm = t*0.75; 
+var bm = b*1.25;
+var rm = r*0.75;
+var lm = l*1.25;
+
```

Comment:
3e) Multiplying is not actually the right way to go here. Imagine that the rectangle has one corner at the origin of the graph (the (0,0) coordinate). In that case, two of your new edges will be in the same places. Also remember that you want thirds, rather than quarters.